### DISCUSSION TITLE
【问题请教】打开打包后的dmg文件时遇到报错（使用的是官方提供的打包示例）：“Weekly”已损坏，无法打开，请教下是否有解决办法？多谢！

### DISCUSSION THREAD

**Discussion by mopduan** [tw93/Pake#664] [Q&A]
【更新】已按：https://github.com/tw93/pake/blob/HEAD/bin/README.md#multi-arch 说明解决了所遇到的问题。

补充说明：
1、使用的打包命令：pake https://weekly.tw93.fun --name Weekly --transparent 
2、本地打包后可以正常打开，但是发给其他人打开时就会报错
3、打包所使用的机器是MBP M1

<img width="278" alt="企业微信截图_654f14ee-99cb-4b33-8442-215a6899d908" src="https://github.com/tw93/Pake/assets/675025/c364fc4e-6ada-4e73-8037-b0bc9901b7ff">

**Comment by tw93** [comment]
假如碰到首次安装不能打开，试试「访达中显示」 然后右键打开即可，假如还是不行请参考 [文档](https://zhuanlan.zhihu.com/p/135948430)

**Reply by NICK-DUAN** [reply]
大佬，问下这是为啥？后面如果正式发包，怎么解决这个问题呢？

**Reply by winter-lau** [reply]
需要 app store 的开发者签名
